update-bot/README.md

```diff
@@ -44,6 +45,19 @@ page uses two MediaWiki templates, [`Fehler_in_Denkmallisten_Tabellenkopf`](wiki
 
 Configuration of the approved templates can be found in the file [`template_config.json`](template_config.json). It configures the names of the approved templates, the name of the ID parameter and a regular expression for validating ID parameters.
 
+
+## commons_bot.py
+Look in Wikimedia Commons in the "Wiki Loves Monuments 2015" category for comments left by the Upload Wizard. Use the comments to update the monument list pages in the German Wikipedia.
```

Comment:
For consistency with descriptions of other bots this sentence could say "Looks in..." and "Uses the comments..."